crates/sui-rpc-benchmark/src/json_rpc/mod.rs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::config::BenchmarkConfig;
+use anyhow::Result;
+use request_loader::load_json_rpc_requests;
+use runner::run_queries;
+use std::time::Duration;
+use tracing::info;
+
+pub mod request_loader;
+pub mod runner;
+
+pub async fn run_benchmark(
+    endpoint: &str,
+    file_path: &str,
+    concurrency: usize,
+    duration_secs: u64,
+) -> Result<()> {
+    let config = BenchmarkConfig {
+        concurrency,
+        duration: Duration::from_secs(duration_secs),
```

Comment:
How is the duration used?

Reply:
good catch, it should have been used but actually not, fixed.